osu.Game/OsuGame.cs

```diff
@@ -464,6 +464,16 @@ public bool OnPressed(GlobalAction action)
                 case GlobalAction.ToggleDirect:
                     direct.ToggleVisibility();
                     return true;
+                case GlobalAction.ToggleMouseButtons:
```

Comment:
we may want to make this binding only work in gameplay screens to avoid confusion.